src/app/pages/services/components/service-nfs/service-nfs.component.spec.ts

```diff
@@ -46,6 +47,11 @@ describe('ServiceNfsComponent', () => {
           '192.168.1.119': '192.168.1.119',
         }),
         mockCall('nfs.update'),
+        mockCall('kerberos.keytab.has_nfs_principal', false),
+        mockCall('directoryservices.get_state', {
+          activedirectory: DirectoryServiceState.Healthy,
+          ldap: DirectoryServiceState.Disabled,
+        }),
```

Comment:
Also needs a test case to check that field appears when conditions are met and that it opens the dialog.